src/main/kotlin/domain/setting/Size.kt

```diff
@@ -0,0 +1,5 @@
+package domain.setting
+
+data class Size(val height: Height, val width: Width) {
+    fun getPossibleLocationCount(): Int = height.value * width.value
```

Comment:
조금 코틀린스럽게 표한한다면 getter의 성격을 띄는 경우 읽기전용 프로퍼티로 표현해보시면 좋습니다.

```suggestion
    val possibleLocationCount: Int = height.value * width.value
```